recommendation/microsoft.owin.hosting.json

```diff
@@ -0,0 +1,126 @@
+﻿{
+  "Name": "Microsoft.Owin.Hosting",
+  "Version": "1.0.0",
+  "Packages": [
+    {
+      "Name": "Owin",
+      "Type": "Nuget"
+    }
+  ],
+  "Recommendations": [
+    {
+      "Type": "Namespace",
+      "Name": "Microsoft.Owin.Hosting",
+      "Value": "Microsoft.Owin.Hosting",
+      "KeyType": "Name",
+      "ContainingType": "",
+      "RecommendedActions": [
+        {
+          "Source": "Amazon",
+          "Preferred": "Yes",
+          "TargetFrameworks": [
+            {
+              "Name": "netcoreapp3.1",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            },
+            {
+              "Name": "net5.0",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            }
+          ],
+          "Description": "Add reference to Microsoft.AspNetCore.Hosting\r\nAdd reference to Microsoft.AspNetCore.Server.Kestrel\r\nRemove Microsoft.Owin.Hosting\r\nRemove Owin\r\n",
+          "Actions": [
+            {
+              "Name": "AddPackage",
+              "Type": "Package",
+              "Value": "Microsoft.AspNetCore.Hosting",
+              "Description": "Add package Microsoft.AspNetCore.Hosting"
+            },
+            {
+              "Name": "AddPackage",
+              "Type": "Package",
+              "Value": "Microsoft.AspNetCore.Server.Kestrel",
+              "Description": "Add package Microsoft.AspNetCore.Server.Kestrel"
+            },
+            {
+              "Name": "AddDirective",
+              "Type": "Using",
+              "Value": "Microsoft.AspNetCore.Hosting",
+              "Description": "Add Microsoft.AspNetCore.Hosting namespace",
+              "ActionValidation": {
+                "Contains": "usingMicrosoft.AspNetCore.Hosting",
```

Comment:
Add a semicolon here. This will make sure it doesn't get a false positive on a child namespace : Microsoft.AspNetCore.Server.Kestrel.*
This applies to all namespace validations

Reply:
Good catch thanks Mark. all done.

Reply:
let's discuss..(Contains is little odd)...

Reply:
As agreed we will revisit possibly using Regex or a count methodology in the future. At the very least spaces will be added back as part of a future PR.